test/test_postman.py

```diff
@@ -216,17 +216,24 @@ def _l1_l2_message_exchange(web3, l1l2_example_contract, l2_contract_address):
     assert balance == 0
 
     # withdraw in l1 and assert contract balance
+    withdraw_amount = 1000
     web3_transact(
         web3,
         "withdraw",
         l1l2_example_contract,
         int(l2_contract_address, base=16),
         USER_ID,
-        1000,
+        withdraw_amount,
     )
 
+    # Check if l2 to l1 message is included in transaction_receipts
+    l2_to_l1_block = get_block(parse=True)
+    assert l2_to_l1_block["transaction_receipts"][0]["l2_to_l1_messages"][0]["payload"][
```

Comment:
I don't think I've ever seen six consecutive square bracket pairs. Could this be split into e.g. three lines?

Reply:
I'll split it into 2 variables

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/330/commits/4869bbce8cdc4b7402a448eb720c7c18d59c1cf1